vertica/README.md

```diff
@@ -0,0 +1,77 @@
+# Agent Check: Vertica
+
+## Overview
+
+This check monitors [Vertica][] through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+The Vertica check is included in the [Datadog Agent][] package.
+
+The user used to connect to the database must be granted the [SYSMONITOR][monitor role] role in order to access the monitoring system tables.
```

Comment:
This should go in the configuration section since the `user` creation is required for the check to work. We could inspire from the MySQL check for the layout/presentation of the process.